db/migrate/20241125060011_add_unique_phase_constraint_to_evaluation_forms.rb

```diff
@@ -0,0 +1,6 @@
+class AddUniquePhaseConstraintToEvaluationForms < ActiveRecord::Migration[7.2]
+  def change
+    remove_index :evaluation_forms, :phase_id, if_exists: true
+    add_index :evaluation_forms, :phase_id, unique: true
```

Comment:
I'm unable to run the migration locally because I have some evaluation_forms that have the same `phase_id`. We'll likely have the same problem trying to deploy this to staging too, so it's probably worth fixing now. Easiest thing is just truncate all the evaluation tables to start over from scratch.
```suggestion
    ActiveRecord::Base.connection.truncate_tables(
      EvaluationForm.table_name,
      EvaluationCriterion.table_name,
      Evaluation.table_name,
      EvaluationScore.table_name
    )
    remove_index :evaluation_forms, :phase_id, if_exists: true
    add_index :evaluation_forms, :phase_id, unique: true
```

Reply:
Yeah. I figured that would happen just wasn't sure how we wanted to approach it. Truncation is fine